index.js

```diff
@@ -39,7 +39,13 @@ module.exports = function(content) {
       JSON.stringify(fileName) +
       ");" +
       "try { global.process.dlopen(module, filePath); } " +
-      "catch(exception) { throw new Error('Cannot open ' + filePath + ': ' + exception); };"
+      "catch(exception) { " +
+      "try { " +
```

Comment:
This should be supported via configuration instead of falling back using `try-catch` statement by default.